lib/linguist/languages.yml

```diff
@@ -2580,6 +2580,17 @@ Git Attributes:
   codemirror_mode: shell
   codemirror_mime_type: text/x-sh
   language_id: 956324166
+Git Commit:
+  type: data
```

Comment:
data, or prose?

Reply:
@Nixinova That's a good question. I'm tentatively leaning towards `data`, for reasons similar to why [`E-mail`](https://github.com/github-linguist/linguist/blob/1d913bbdff8e31f1dcf0ba6ce482f4edb5715fc7/lib/linguist/languages.yml#L1777-L1791) is categorised as `data`: it's a transient data format that, while user-readable, is ultimately destined for processing by a program instead of presented as-is.

That's an unbelievably shaky justification, but classifying Git commit-messages as `prose` feels… wrong, somehow.